vendors/twtg/profiles/twtg_RFGroup1_1.0.4a_RP2-1.0.3_classA.yaml

```diff
@@ -0,0 +1,98 @@
+# SHALL Follow this convention: <vendorID> (lowercase) + '_' + <Profile Group> + '_' <LoRaWAN L2 & PHY versions> + '_class' + <LoRaWAN class A, B or C>
+# Follow the information below to know the profile group to use in the ID
+#    Profile Group
+#    RFGroup1:  eu863-870, kr920-923, in865-867, ru864-870
+#    RFGroup2:  us902-928, cn470-510, au915-928 (only for LoRaWAN 1.0.2revB or below)
+#    RFGroup3:   au915-928 (starting from LoRaWAN 1.0.2revC)
+#    RFGroup4: as923
+# Please do not change the existing IDs (Some may follow the old format)
+# The name of the profile .yaml file should be the same as the following ID
+id: twtg_RFGroup1_1.0.4a_RP2-1.0.3_classA
+# LoRaWAN class of the Device: Possible values: [ A, B, C ] (Should not have several values)
+# 'A': Class A (Bi-directional end-devices with downlink listening only after uplink transmission)
+# 'B': Class B (Bi-directional end-devices with downlink listening on predefined synchronized pingslots)
+# 'C': Class C (Bi-directional end-devices with permanent downlink listening)
+loRaWANClass: A
+# Comma-separated list of regional profiles supported by this device. Possible values are eu868, us915, as923, au915, eu433, in865, kr920, ru864, cn470.
+ISMbands: eu868,in865,kr920
+# Typical mobility profile of the device. Possible values are 'near_static' (also valid for static devices), 'walking_speed', 'vehicular_speed'
+# or 'random' (not known, changes over time).
+# The motion indication property will not be taken into consideration here if it is overwritten in the model.yaml
+motionIndicator: NEAR_STATIC
+
+mac:
+    # Activation modes supported by the device (put true where applicable).
+    # (ABP)
+    activationByPersonalization: false
+    #(OTAA)
+    overTheAirActivation: true
+    # Supported version of the LoRaWAN MAC (Layer-2) specification. Possible values are '1.0.0/1.0.1', '1.0.2', '1.0.3', '1.0.4', '1.1'
+    loRaWANMacVersion: 1.0.4
+    # Supported version of the LoRaWAN regional parameters (PHY) specification. Possible values are '1.0.2revA', '1.0.2revB', '1.0.2revC', 
+    # '1.0.3revA', 'RP2-1.0.0', 'RP2-1.0.1', 'RP2-1.0.2', 'RP2-1.0.3', 'RP2-1.0.4', 'RP2-1.0.5'.
+    regionalParameterVersion: RP2-1.0.3
+
+    # The device Tx Power Capabilities here will not be taken into consideration if it is overwritten in the model.yaml
+    devicesTxPowerCapabilities:
+        # Minimum device TX Conducted output power in dBm.
+        minTxPower:
+        # Maximum device TX Conducted output power in dBm.
+        maxTxPower: 14
+        # Minimum device TX Radiated output power in dBm.
+        minTxEIRP:
+        # Maximum device TX Radiated output power in dBm.
+        maxTxEIRP: 14
+
+    uplinkFrameRepetition: 
+        # Maximum number of transmissions per uplink frame, supported by the device. 
+        maxNbTrans:
+        # How many uplink transmissions does the device use in Confirmed mode if it doesn't receive any DL ACK?
+        maxRedundancyForConfirmedUL: 0
```

Comment:
For what this information is used? We have one product that has currently `maxTxPower 14`, but there are some devices in the field of an early batch (with the same functionality) that have a lower `maxTxPower`. Should we make different profiles for that? For the customer there is no different identification on the device, so it wouldn't know which profile it should use. If we just use a profile with `maxTxPower=14` on a device with a lower `maxTxPower`, what would go wrong?

Reply:
The maxTxPower is meant to make our network respect the imposed regional legal limitations.
Can you tell me what the maxTxPower of the first batch is please? Ideally there should be 2 profiles, but this would most likely create confusion for users. If the difference is small, then you should stick with one profile and have the higher level of the maxTxPower (14).